grpc/tests/cmd_run_test.go

```diff
@@ -0,0 +1,132 @@
+package tests
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/grafana/xk6-grpc/grpc"
+	"github.com/grafana/xk6-grpc/grpc/testutils"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.k6.io/k6/cmd"
+	k6Tests "go.k6.io/k6/cmd/tests"
+	"go.k6.io/k6/errext/exitcodes"
+	"go.k6.io/k6/js/modules"
+)
+
+func init() {
+	modules.Register("k6/x/grpc", new(grpc.RootModule))
```

Comment:
nit: this can likely be done by just importing the package that does this already - in this case `github.com/grafana/xk6-grpc`

Reply:
:+1: Resolved with the latest force push